packages/okta-react-native/index.js

```diff
@@ -49,7 +62,44 @@ export const createConfig = async({
   );
 } 
 
-export const signIn = async() => {
+export const getAuthClient = () => {
+  if (!authClient) {
+    throw { 
+      code: "-100", 
+      message: 'OktaOidc client isn\'t configured, check if you have created a configuration with createConfig' 
+    };
+  }
+  return authClient;
+}
+
+export const signIn = async(options) => {
+  // Custom sign in
+  if (options && typeof options === 'object') {
+    return authClient.signIn(options)
+      .then((transaction) => {
+        const { status, sessionToken } = transaction;
+        if (status !== 'SUCCESS') {
+          throw new Error('Transaction status other than "SUCCESS" has been return, please handle it properly by calling "authClient.tx.resume()"');
+        } 
+        return authenticate({ sessionToken });
+      })
+      .then(token => {
+        if (!token) {
+          throw new Error('Failed to get accessToken');
+        }
+
+        return token;
+      })
+      .catch(error => {
+        throw {
```

Comment:
should we be throwing a `new Error` object here? If a bare object is thrown there will be no stack trace.